README.md

```diff
@@ -207,4 +207,12 @@ Overrides the console.log input annd prints it in the mOcKiNgCaSe.
 
 <hr>
 
+## CDN
+Mockingcase can be served to the browser from a CDN. Simply put `index.js` into your server files or use Mockingcase from a CDN. Some CDNs are provided below:
```

Comment:
> Simply put `index.js` into your server files

What do you mean?

Reply:
Instead of pulling Mockingcase from a CDN they would host it themselves on their server. I'll refactor that statement.